app/admin/litigations.rb

```diff
@@ -113,6 +112,46 @@
 
   form partial: 'form'
 
+  batch_action :publish,
```

Comment:
See a potential here to refactor those to some kind of DSL extensions that we have. But could be done in another PR.